lib/registries/addon/overview/route.ts

```diff
@@ -72,8 +74,14 @@ export default class Overview extends GuidRoute {
                 ),
                 institution: (institutions as SparseModel[]).map(institution => institution.name as string),
             };
-
-            this.set('headTags', this.metaTags.getHeadTags(metaTagsData));
+            const headTags = [...this.metaTags.getHeadTags(metaTagsData), {
+                type: 'link',
+                attrs: {
+                    rel: 'icon',
+                    href: provider.assets!.favicon,
```

Comment:
is it safe to assume that `assets` will always be defined? Also if `assets` is defined, but favicon isn't, what would the repercussions of an `undefined` href be here?